src/app/app.module.ts

```diff
@@ -32,26 +34,31 @@ import { MyApp } from './app.component';
     SearchPipe,
     ShowPictureComponent,
     TabsPage,
-    VerificationPage
+    VerificationPage,
```

Comment:
I added `npm run ionic:lint` which executes the linter, please add the rules you want...